### ISSUE TITLE
Is the source of https://vps.zerodream.net/ available?

### ISSUE THREAD

**Issue by purerosefallen** [kasuganosoras/Libvirt-Manager#1]
I just saw an `example.php`, but nothing else related to web content...
May I ask where I can get the source of your website? I'd love to have a try of this platform on my own dedis. Thanks.

**Comment by YuyuYuna** [comment]
It is in src folder

**Comment by kasuganosoras** [comment]
Yes, https://vps.zerodream.net/ is based on this project. There is currently no open source.
Now you can quickly deploy Libvirt-Manager with composer and integrate it into your project.
